runtests

```diff
@@ -81,6 +85,9 @@ runtest() {
   else
     echo -e "<<<test end, result: $result, duration: ${duration}s>>\n"
   fi
+  case_result=""
```

Comment:
better to put the definition of the variable at the top of the function.

Reply:
It's already defined on top of the function:  "local case_result="""
Here  case_result=""  is cleaning the left result before get new case_result for  loop  usage,  and there is loop here.
So this line could be removed.
Thanks!

Reply:
Got it. Thanks for the explanation.